.github/workflows/testing.yaml

```diff
@@ -0,0 +1,248 @@
+          chmod a+x installer.sh
+          sudo ./installer.sh
+
+      - name: Install tarantool ${{ matrix.tarantool }}
+        run: |
+          # Install tarantool.
+          #
+          # We don't use tarantool/setup-tarantool GitHub Action
+          # at the moment due to several reasons:
+          #
+          # 1. No way to install a non-last tarantool version from
+          #    a repository:
+          #    https://github.com/tarantool/setup-tarantool/issues/15
+          # 2. No way to install a live package:
+          #    https://github.com/tarantool/setup-tarantool/issues/9
+          # 3. We likely will install tarantool in a container
+          #    job and it'll require support from the action side:
+          #    https://github.com/tarantool/setup-tarantool/issues/11
+          sudo apt-get install -y "tarantool=${T_VERSION}*" "tarantool-dev=${T_VERSION}*"
+
+      - name: Verify tarantool version
+        run: |
+          # Workaround https://github.com/tarantool/tarantool/issues/4983
+          # Workaround https://github.com/tarantool/tarantool/issues/5040
+          tarantool -e "require('fiber').sleep(0) assert(_TARANTOOL:startswith('${T_VERSION}'), _TARANTOOL) os.exit()"
+
+      - name: Install build dependencies for the module
+        run: sudo apt-get install -y libcurl4-openssl-dev
+        if: '!matrix.dont_install_system_libcurl_header'
+
+      - name: Inject an error into system's curl/curl.h
+        run: |
+          # Note: It hardcodes known libcurl headers location on
+          # Ubuntu Focal.
+          CURL_H=/usr/include/x86_64-linux-gnu/curl/curl.h
+          echo '#error Deliberately broken from CI' | sudo tee "${CURL_H}"
+        if: matrix.break_system_libcurl_header
+
+      - name: Clone the module
+        uses: actions/checkout@v2
+
+      - name: Install requirements
+        run: |
+          tarantoolctl rocks install luatest 0.5.0
+          tarantoolctl rocks install luacheck 0.25.0
+          tarantoolctl rocks install cartridge 2.5.1
+
+      - name: Run luatest tests
+        run: .rocks/bin/luatest -v
+
+
+  testing_mac_os:
+    # We want to run on external PRs, but not on our own internal
+    # PRs as they'll be run by the push to the branch.
+    #
+    # The main trick is described here:
+    # https://github.com/Dart-Code/Dart-Code/pull/2375
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on:
+          - macos-10.15
+          - macos-11.0
+        tarantool:
+          - brew
+#          - 1.10.7 # tarantoolctl didn't find
+#          - 1.10.8
+#          - 1.10.9
+#          - 1.10.10
+#          - 2.4.1
+#          - 2.4.2
+#          - 2.4.3
+#          - 2.5.1
+#          - 2.5.2
+#          - 2.5.3
+#          - 2.6.1
+#          - 2.6.2
+#          - 2.6.3
+#          - 2.7.1
+#          - 2.7.2
+#          - 2.8.1
+#          - master
```

Comment:
Use `-DENABLE_DIST=ON` cmake option to install tarantoolctl into a destination directory.

Reply:
Also we have to add LUA_PATH to ENV and pass TARANTOOL_DIR to tarantoolctl rocks install, but there will be problems like we did when installing cartridge(http), see https://github.com/tarantool/expirationd/blob/9edcbfd10f77680befdaf4b2a49b115702f2e7d9/.github/workflows/testing.yaml#L223-L224

Reply:
moved it to another PR #55